src/com/projectkorra/projectkorra/util/DamageHandler.java

```diff
@@ -90,9 +100,20 @@ public static void damageEntity(final Entity entity, Player source, double damag
 		}
 
 	}
-
+	
+	public static boolean hasArmor(LivingEntity entity) {
+		if(entity.getEquipment().getBoots()!=null || entity.getEquipment().getChestplate()!=null || entity.getEquipment().getHelmet()!=null || entity.getEquipment().getLeggings()!=null) {
+			return true;
+		}
+		return false;
+	}
+	
+	public static Map<LivingEntity, Long> CooldownDamage() {
```

Comment:
In regards to Java standards, this should be `getCooldownDamageMap()` or something with camelcasing